docs/get-started/scaffolding.md

```diff
@@ -0,0 +1,26 @@
+---
+id: scaffolding
+title: Scaffolding
+sidebar_label: Scaffolding
+---
+
+# Webiny Scaffold CLI
+
+### Introduction
+Sometimes when you develop applications with Webiny you will end up spending time doing common tasks that can easily become repetitive and tedious over time. This is where `webiny scaffold` comes into play.
+
+### Usage
+This assumes you already have a Webiny project to work on. If not, see the [getting started guide](/docs/get-started/quick-start).
+
+After running `webiny scaffold`, you will be prompted to choose from various templates. By default, Webiny comes with a couple of them, but you can create your own, or install the existing ones from NPM.
+
+After picking one of them, you might be asked a couple more questions in order to bootstrap your template. For example, a GraphQL Apollo Service template could ask for the name of your service:
+```
+? The name of your service
+```
+
+Once that's finished, your template should have been generated succesfully. As simple as that!
+
+### Default Templates
+* GraphQL Apollo Service (in PR)
```

Comment:
You can skip these `(in PR)` and `WIP` since we know you're working on it, and these docs won't be published until those packages are available on npm. So feel free to type stuff in the docs that is not yet implemented, if it's something you're already working on. This way you won't have to clean it up later.

Reply:
ah, great to know

done